contract/models/account_analytic_account.py

```diff
@@ -7,6 +7,7 @@
 
 from dateutil.relativedelta import relativedelta
 
+import datetime as date
```

Comment:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#imports

I think we should have a extra line between library imports and odoo imports